src/main/resources/earlierversions/detect7.sh

```diff
@@ -2,6 +2,17 @@
 
 echo "Detect Shell Script ${SCRIPT_VERSION}"
 
+write_deprecation_msg() {
+	echo "***"
+	echo "***"
+	echo "*** Warning: Detect 7 is End of Service from 1st March 2024, this script will be removed and will no longer be accessible."
```

Comment:
. This script will no longer be available.

or

and this script will no longer be available.